python/tchannel/tornado/util.py

```diff
@@ -52,3 +53,15 @@ def print_arg(request, index):
                 print chunk
     except Exception:
         pass
+
+
+def deserialize_json(content):
+    if content is None:
+        return ""
+    return json.loads(content)
+
+
+def serialize_json(jobject):
+    if jobject is None:
+        return {}
```

Comment:
The result of serializing `None` is not an empty dictionary. `serialize_json` must always return a string.